src/status_im/chat/views/toolbar_content.cljs

```diff
@@ -4,8 +4,8 @@
             [clojure.string :as str]
             [cljs-time.core :as t]
             [status-im.ui.components.react :refer [view
-                                                text
```

Comment:
Maybe we could use namespace here?

Reply:
it's just cursive, not related to this PR

Reply:
I know but it's an opportunity to move to `:as react` :)

Reply:
okay

Reply:
there are not only react, don't want to mess it in this PR